demisto_sdk/commands/prepare_content/preparers/incident_to_alert.py

```diff
@@ -0,0 +1,260 @@
+}
+
+
+"""
+PLAYBOOK HELPER FUNCTIONS
+"""
+
+
+def prepare_descriptions_and_names_classifier(
+    name_or_description_content: str, replace_incident_to_alert: bool
+):
+    if replace_incident_to_alert:
+        name_or_description_content = edit_names_and_descriptions_for_playbook(
+            name_or_description_content, replace_incident_to_alert
+        )
+    return edit_names_and_descriptions_for_playbook(
+        name_or_description_content, False
+    )  # Here remove the wrapper
+
+
+def prepare_descriptions_and_names(
+    data: dict, marketplace: MarketplaceVersions
+) -> dict:
+    # Replace incidents to alerts only for XSIAM
+    replace_incident_to_alert = marketplace == MarketplaceVersions.MarketplaceV2
+
+    # Descriptions and names for all tasks
+    for task_key, task_value in data.get("tasks", {}).items():
+
+        if description := task_value.get("task", {}).get("description", ""):
+            if description != "commands.local.cmd.set.incident":
+                # Since it is a server key, we do not want to change it
+                data["tasks"][task_key]["task"][
+                    "description"
+                ] = prepare_descriptions_and_names_classifier(
+                    description, replace_incident_to_alert
+                )
+
+        if name := task_value.get("task", {}).get("name", ""):
+            data["tasks"][task_key]["task"][
+                "name"
+            ] = prepare_descriptions_and_names_classifier(name, replace_incident_to_alert)
+
+    # The external playbook's description
+    if description := data.get("description"):
+        data["description"] = prepare_descriptions_and_names_classifier(
+            description, replace_incident_to_alert
+        )
+
+    # The external playbook's name
+    if name := data.get("name"):
+        data["name"] = prepare_descriptions_and_names_classifier(
+            name, replace_incident_to_alert
+        )
+
+    return data
+
+
+def edit_names_and_descriptions_for_playbook(
+    name_or_description_field_content: str, replace_incident_to_alert: bool
+) -> str:
+    if replace_incident_to_alert:
+        replacements = NOT_WRAPPED_RE_MAPPING
+    else:
+        replacements = WRAPPED_MAPPING
+
+    new_content = name_or_description_field_content
+
+    for pattern, replace_with in replacements.items():
+        new_content = re.sub(pattern, replace_with, new_content)
+
+    return new_content
+
+
+def replace_playbook_access_fields_recursively(
+    datum: Any, replaceable_scripts: Tuple[str, ...]
+) -> Any:
+    if isinstance(datum, list):
+        return [
+            replace_playbook_access_fields_recursively(item, replaceable_scripts) for item in datum
+        ]
+
+    elif isinstance(datum, dict):
+        for key, val in datum.items():
+            if isinstance(val, str):
+                if key in {"root", "simple"} and "incident" in val:
+                    val = re.sub(
+                        r"(?<!\.)\bincident\b", "alert", val
```

Comment:
why the first `\b`?

Reply:
@darkushin can you please look at it

Reply:
The two `\b` are to mark the word boundary of `incident` so that if a substring of is given it won't be considered as a match.
If you have a better regex I will be happy to change :)